app/src/androidTest/java/com/soen490chrysalis/papilio/LoginActivityUITest.kt

```diff
@@ -1,48 +1,24 @@
 package com.soen490chrysalis.papilio
 
-import android.app.Activity
```

Comment:
This file has too many deletions and changes in general. This is suspicious because it should not have been touch on a branch about "user profile". The only changes that should be there is the scrollTo() fix that I added yesterday. 
When I compare this file with the one on the main branch, there are several test methods missing. This definitely needs an update!